Automattic/jetpack
# Custom Content Types: fix PHPCS errors in Nova CPT file.

See #21963

#### Changes proposed in this Pull Request:

* Fix PHPCS linting errors in Nova CPT.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

**Requirements**

**On WordPress.com simple**

Enable the Confit theme.

**On self-hosted**

Add the following snippet to your theme's `functions.php` file, or to a functionality plugin:
```php
// Enable Nova CPT.
add_action( 'after_setup_theme', function () {
	add_theme_support( 'nova_menu_item' );
} );
```

**To test**

- Go to the new menu, and add a new menu item.
- Make sure you can add labels as well as prices, and then publish that menu item.
- Make sure you can edit existing menus as well.
- After adding a few items (make sure you place them in different menu sections, e.g. entrees, mains, ...), you should see them displayed nicely in the "Menu items" menu.
- You should be able to drag and drop items to reorder them, and then click the "Save New Order" button, and have those changes saved.
- Then go to the "Add many items" menu, and ensure this works too.
